packages/synapse-react-client/src/components/AccessRequirement/ImposeRestrictionDialog/ImposeRestrictionDialog.tsx

```diff
@@ -41,8 +43,8 @@ export default function ImposeRestrictionDialog(
 
   const { entityId, open, onClose } = props
   const [isSensitiveHumanData, setIsSensitiveHumanData] = React.useState<
-    boolean | undefined
-  >(undefined)
+    boolean | null
+  >(null)
```

Comment:
Start with `null` to prevent warning about input being uncontrolled because we start with `undefined` value.